### PR TITLE
fix: Updates regular expressions naming in recognizer types 

### LINKED ISSUE #4060: [UX] Recognizer type's naming rule needs to be consistent
## Describe the bug

image

### PR DESCRIPTION
**Pull request by stevkan**
## Description

The naming convention between the different recognizer types is inconsistent when the user clicks on the associated dropdown. These changes are implemented to bring consistency across various JSON, .spec.ts, .dialog, and .md files.

## Task Item

Closes #4060

## Screenshots

image

### CODE REVIEW

File: docs/archived docs/tutorial/tutorial-add-dialog.md [hunk 1]
```diff
@@ -59,18 +59,18 @@ You can break pieces of your conversation flow into different dialogs and can ch
...
-   ![regular expression recognizer](../media/tutorial-weatherbot/02/recognizer-type.png)
+   ![regular expressions recognizer](../media/tutorial-weatherbot/02/recognizer-type.png)
 
 4. Enter **weather** for both **Intent** and **Pattern**. Make sure you press **Enter** in your keyboard to save the setting.
 
    ![intent and pattern](../media/tutorial-weatherbot/02/intent-pattern.png)
 
    > [!NOTE]
-   > This tells the bot to look for the word "weather" anywhere in an incoming message. Regular expression patterns are generally much more complicated, but this is adequate for the purposes of this example.
+   > This tells the bot to look for the word "weather" anywhere in an incoming message. Regular expressions patterns are generally much more complicated, but this is adequate for the purposes of this example.
```
Comment (a-b-r-o-w-n) [inline docs/archived docs/tutorial/tutorial-add-dialog.md:73]:
```suggestion
   > This tells the bot to look for the word "weather" anywhere in an incoming message. Regular expression patterns are generally much more complicated, but this is adequate for the purposes of this example.
```

File: docs/olddocs/triggers_and_events.md [hunk 2]
```diff
@@ -84,7 +84,7 @@ Extracted entities are passed along to any triggered actions or child dialogs us
 
 When triggered, if LUIS is able to identify a city, the city name will be made available as `@city` within the triggered actions. The entity value can be used directly in expressions and LG templates, or [stored into a memory property](using_memory.md) for later use.
 
-### Regular Expression Recognizer
+### Regular expressions Recognizer
```
Comment (a-b-r-o-w-n) [inline docs/olddocs/triggers_and_events.md:87]: The casing looks weird here.

**Review by a-b-r-o-w-n — CHANGES_REQUESTED** [review]
I'll have to say that "regular expressions recognizer" doesn't seem as correct as "regular expression recognizer". @zxyanliu how did we arrive at using the plural to describe the recognizer?

**Comment by zxyanliu** [comment]
> I'll have to say that "regular expressions recognizer" doesn't seem as correct as "regular expression recognizer". @zxyanliu how did we arrive at using the plural to describe the recognizer?
@a-b-r-o-w-n we did not arrive at using the plural to describe the recognizer, I shared my opinion on the casing in this issue: 
https://github.com/microsoft/BotFramework-Composer/issues/4060#issuecomment-700305574

File: docs/archived docs/tutorial/tutorial-add-dialog.md (hunk 1 above)
Comment (zxyanliu) [reply docs/archived docs/tutorial/tutorial-add-dialog.md:73]: These docs are outdated and archived, I am not sure if we still need to update them.

File: docs/olddocs/triggers_and_events.md (hunk 2 above)
Comment (zxyanliu) [reply docs/olddocs/triggers_and_events.md:87]: Yes, the casing is wired. 
These docs are outdated and archived, I am not sure if we still need to update them.

File: docs/archived docs/tutorial/tutorial-add-dialog.md (hunk 1 above)
Comment (stevkan) [reply docs/archived docs/tutorial/tutorial-add-dialog.md:73]: Yes, you are right. I was a bit _too_ heads down and missed that this is an archived doc. I will revert.

File: docs/olddocs/triggers_and_events.md (hunk 2 above)
Comment (stevkan) [reply docs/olddocs/triggers_and_events.md:87]: Again, a little too focused, and the error somehow snuck buy. Regardless, I will revert it.

**Comment by stevkan** [comment]
Moving from singular to plural was something I had suggested as a consideration. It looks like @zxyanliu picked up on it, unintentionally, which I then took as confirmation.

I'll correct those instances back to singular that should be.

File: Composer/cypress/integration/TriggerCreation.spec.ts [hunk 3]
```diff
@@ -10,7 +10,7 @@ context('Creating a new trigger', () => {
   it('can create different kinds of triggers ', () => {
     cy.visitPage('Design');
     cy.findByTestId('recognizerTypeDropdown').click();
-    cy.findByText('Regular Expression').click();
+    cy.findByText('Regular expressions recognizer').click();
```
Comment (zxyanliu) [inline Composer/cypress/integration/TriggerCreation.spec.ts:13]:
```suggestion
    cy.findByText('Regular expression recognizer').click();
```

File: Composer/packages/server/src/locales/en-US.json [hunk 4]
```diff
@@ -1748,8 +1748,8 @@
   "regex_intent_is_already_defined_df095c1f": {
     "message": "RegEx { intent } is already defined"
   },
-  "regular_expression_855557bf": {
-    "message": "Regular Expression"
+  "regular_expressions_recognizer_c11bbdda": {
+    "message": "Regular expressions recognizer"
```
Comment (zxyanliu) [inline Composer/packages/server/src/locales/en-US.json:1752]:
```suggestion
    "message": "Regular expression recognizer"
```

File: Composer/packages/ui-plugins/composer/src/defaultRecognizerSchema.ts [hunk 5]
```diff
@@ -14,7 +14,7 @@ const FallbackRecognizerJsonEditor: RecognizerOptions = {
 
 export const DefaultRecognizerSchema: RecognizerUISchema = {
   [SDKKinds.RegexRecognizer]: {
-    displayName: () => formatMessage('Regular Expression'),
+    displayName: () => formatMessage('Regular expressions recognizer'),
```
Comment (zxyanliu) [inline Composer/packages/ui-plugins/composer/src/defaultRecognizerSchema.ts:17]:
```suggestion
    displayName: () => formatMessage('Regular expression recognizer'),
```

**Comment by zxyanliu** [comment]
Hi @stevkan I left several comments, also we do not need to update the docs since they were achieved. 
Other than those, the PR looks good to me. Thanks!

**Pull request closed without merging**